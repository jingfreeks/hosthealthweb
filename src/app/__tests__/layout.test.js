import { render, screen } from "@testing-library/react";
import Layout from "../layout";


// Mock useRouter:
jest.mock("next/navigation", () => ({
    useRouter() {
      return {
        prefetch: () => null
      };
    }
  }));
describe("Layout ", () => {
  it("renders Layout unchanged to match snapshot", () => {
    const { container } = render(<Layout />);
    expect(container).toMatchSnapshot();
  });
});
