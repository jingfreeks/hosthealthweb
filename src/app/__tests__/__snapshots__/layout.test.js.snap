// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Layout  renders Layout unchanged to match snapshot 1`] = `
<div>
  <html
    lang="en"
  >
    <body
      class="bg-slate-800"
    >
      <nav
        class="bg-slate-600 p-4 flex justify-between flex-col md:flex-row sticky top-0 drop-shadow-xl"
      >
        <h1
          class="text-3xl font-bold text-white grid place-content-center mb-2 md:mb-0"
        >
          <a
            href="/"
          >
            Hosthealth
          </a>
        </h1>
        <form
          class="w-50 flex justify-center md:justify-between"
        >
          <input
            class="bg-white p-2 w-80 text-xl rounded-xl"
            placeholder="Search"
            type="text"
            value=""
          />
          <button
            class="p-2 text-xl rounded-xl bg-slate-300 ml-2 font-bold"
          >
            üîç
          </button>
        </form>
      </nav>
      <nav
        class="bg-slate-600  flex justify-end flex-col md:flex-row sticky top-0 drop-shadow-xl"
      >
        <h2
          class="text-1xl font-bold text-white grid place-content-center mb-2 md:mb-0 p-4"
        >
          <a
            href="/"
          >
            Home
          </a>
        </h2>
        <h2
          class="text-1xl font-bold text-white grid place-content-center mb-2 md:mb-0 p-4"
        >
          <a
            href="/"
          >
            News
          </a>
        </h2>
        <h2
          class="text-1xl font-bold text-white grid place-content-center mb-2 md:mb-0 p-4"
        >
          <a
            href="/"
          >
            Contact
          </a>
        </h2>
        <h2
          class="text-1xl font-bold text-white grid place-content-center mb-2 md:mb-0 p-4"
        >
          <a
            href="/"
          >
            About
          </a>
        </h2>
        <h2
          class="text-1xl font-bold text-white grid place-content-center mb-2 md:mb-0 p-4"
        >
          <a
            href="/"
          >
            Sign-Out
          </a>
        </h2>
      </nav>
    </body>
  </html>
</div>
`;
